### PR TITLE
Create Simple Chat App Update

### LINKED ISSUE #1146: Update docs to reflect  Relay V1 (server) removal from go-ipfs
We need to  update all places that mention `EnableRelayHop` –   Relay V1 server was removed from go-ipfs 0.11 and now requires running a separate binary.

See details in:
-   [go-ipfs v0.11.0 Release Notes](https://github.com/ipfs/go-ipfs/releases/tag/v0.11.0)
  - Relay V1 can be still provided by this standalone binary: https://github.com/libp2p/go-libp2p-relay-daemon#minimal-config-file / https://dist.ipfs.io/#libp2p-relay-daemon

The most important is to update "browsers connectivity" tutorial by @TheDiscordian: 

1. https://blog.ipfs.io/2021-06-10-guide-to-ipfs-connectivity-in-browsers/
2. https://docs.ipfs.io/how-to/create-simple-chat-app/

(We seem to already document v1 and v2 in https://docs.ipfs.io/concepts/nodes/#types)

### PR DESCRIPTION
**Pull request by TheDiscordian**
Updated diagram, changed links to be up-to-date, removed Docker section, added warning about circuit-relay v1 removal from go-ipfs & linked to a repo with most of the code needed to use the chat app /w a relay (but missing pubsub & the ability to list addrs, so no announcing) <del>fixes</del>  towards  #1146

This simply makes the guide correct again, however doesn't instruct the user on how to setup a circuit-relay anymore. The reason for this being is [go-libp2p-relay-daemon](https://github.com/libp2p/go-libp2p-relay-daemon) doesn't currently have any way to:

1. Announce over pubsub
2. List connected addresses

So unless I missed something, while it's awesome having most of the hard work needed done, it's still not enough to bring life to using circuit-relay in a browser right now, not that I can figure out anyways. In the future I or someone else can likely take the [Python script](https://gist.github.com/TheDiscordian/51962fea72f8d5a5c3bba79dd7009e1c) which is used for advertising currently, and build a custom libp2p node that bundles the pubsub announces right into it, which would completely solve the problem.

### CODE REVIEW

File: docs/how-to/create-simple-chat-app.md [hunk 1]
```diff
@@ -105,10 +103,14 @@ ipfs = await Ipfs.create({
...
+Please note that this how-to uses example star nodes — however, those won't necessarily always be accessible. Currently it's important to either find a reliable star node or host your own. You can host your own by following the instructions for a [native setup](https://github.com/libp2p/js-libp2p-webrtc-star/tree/master/packages/webrtc-star-signalling-server) or for a [Docker container](https://github.com/libp2p/js-libp2p-webrtc-star/blob/master/packages/webrtc-star-signalling-server/DEPLOYMENT.md) which includes Nginx (for SSL). If you opt for the native setup, we cover the Nginx reverse proxy process and SSL cert retrieval later in this post.
 
 ### p2p-circuit
 
+:::warning
+This section is currently only relevant for go-ipfs versions **before** v0.11.0 as it's about circuit relay v1. There is currently no solution available to directly replace it, though most of the work has been complete [here](https://github.com/libp2p/go-libp2p-relay-daemon).
```
Comment (johnnymatthews) [inline docs/how-to/create-simple-chat-app.md:111]:
```suggestion
This section is currently only relevant for go-ipfs versions **before** v0.11.0 as it's about circuit relay v1. There is currently no solution available to directly replace it, though most of the work [has been completed](https://github.com/libp2p/go-libp2p-relay-daemon).
```

**Review by johnnymatthews — APPROVED** [review]
Looks good. Merge when ready.

File: docs/how-to/create-simple-chat-app.md [hunk 2]
```diff
@@ -105,10 +103,14 @@ ipfs = await Ipfs.create({
...
+Please note that this how-to uses example star nodes — however, those won't necessarily always be accessible. Currently it's important to either find a reliable star node or host your own. You can host your own by following the instructions for a [native setup](https://github.com/libp2p/js-libp2p-webrtc-star/tree/master/packages/webrtc-star-signalling-server) or for a [Docker container](https://github.com/libp2p/js-libp2p-webrtc-star/blob/master/packages/webrtc-star-signalling-server/DEPLOYMENT.md) which includes Nginx (for SSL). If you opt for the native setup, we cover the Nginx reverse proxy process and SSL cert retrieval later in this post.
 
 ### p2p-circuit
 
+:::warning
+This section is currently only relevant for go-ipfs versions **before** v0.11.0 as it's about circuit relay v1. There is currently no solution available to directly replace it, though most of the work [has been completed](https://github.com/libp2p/go-libp2p-relay-daemon).
```
Comment (lidel) [inline docs/how-to/create-simple-chat-app.md:111]: @TheDiscordian was there a hard blocker that made you write this? iiuc configuring it to be `RelayV1: true`  should be drop-in replacement, no?

```suggestion
This section is currently only relevant for go-ipfs versions **before** v0.11.0 as circuit relay v1 server is no longer provided with go-ipfs. There is currently a standalone solution available  [go-libp2p-relay-daemon](https://github.com/libp2p/go-libp2p-relay-daemon#readme) configured as `RelayV1: true`.
```

**Review by lidel — CHANGES_REQUESTED** [review]

**Review by lidel — APPROVED** [review]
This is fine as band-aid (adds note that go-ipfs 0.11 needs to be used), but does not remove  references to deprecated `EnableRelayHop` from our docs.

We should keep  #1146 open until we remove all mentions.

**Pull request merged**